### DISCUSSION TITLE
Custom Button Icons

### DISCUSSION THREAD

**Discussion by tpaolillo** [arkypita/LaserGRBL#1093] [Q&A]
Where do you guys get your custom button icons? I cant seem to even unzip the GZ files.

Thanks
Tony

**Comment by StuartB4** [comment]
You don't need to unzip them. Use the compressed file as it is. Just do this. 
![78787878787](https://user-images.githubusercontent.com/16980026/89745155-b806c100-daa9-11ea-8fef-8bff3558b0b5.jpg)

Click Yes to the ones you want or No to the ones you don't want.

**Reply by crystale1984** [reply]
When I right click on the blank area of the bar nothing happens.  What do I do?  Everything is connected and I'm a newbie so I am still trying to setup.

**Comment by tpaolillo** [comment]
I know how to import them. My question was about the icons. How do I extract or get more icons.

**Comment by StuartB4** [comment]
Just make your own with a graphics app, then load them in to the required button.
![33333333333333](https://user-images.githubusercontent.com/16980026/89746579-6dd60d80-dab2-11ea-89a4-5a62836a272e.jpg)

**Comment by arkypita** [comment]
I think I have shared my icon set in some other post. Get a look here: https://github.com/arkypita/LaserGRBL/issues/306
